### PR TITLE
feat: Multi Button: Added option to add Icon.

### PR DESCRIPTION
**Pull request by vrundakansara**
### Description

1. Improvement: Multi Button: Added option to add Icon. [[Trello Task](https://trello.com/c/Pb1wETpD/61-multibutton-add-option-to-add-icons-in-the-button)]
2. Improvement: Post Layout: Added `post_class()` compatibility to Post Grid, Post Masonry, and Post Carousel.

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
1. Icon option [Screenshot](https://share.getcloudapp.com/lluJYKDo)
2. `post_class()` [Ticket](https://wordpress.org/support/topic/override-render_image-function/)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [x] My code has proper inline documentation
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/buttons-child/edit.js [hunk 1]
```diff
@@ -41,11 +46,16 @@ class UAGBButtonsChild extends Component {
...
 		this.onClickLinkSettings = this.onClickLinkSettings.bind(this)
 		this.onChangeOpensInNewTab = this.onChangeOpensInNewTab.bind(this)
+		this.setIcon = this.setIcon.bind(this)
 		this.state = {
 			isURLPickerOpen:false,
 		}
 	}
 
+	setIcon( value ) {
+		this.props.setAttributes( { icon: value } )
```
Comment (RajkiranBagal) [inline src/blocks/buttons-child/edit.js:56]: Any reason to create a separate function for this?

File: src/blocks/buttons-child/edit.js [hunk 2]
```diff
@@ -152,6 +174,30 @@ class UAGBButtonsChild extends Component {
 						checked={ inheritFromTheme }
 						onChange={ ( value ) => setAttributes( { inheritFromTheme: ! inheritFromTheme } ) }
 					/>
+					<h2>{ __( "Button Icon" ) }</h2>
+					<FontIconPicker {...icon_props} />
+					{ icon != "" &&
```
Comment (RajkiranBagal) [inline src/blocks/buttons-child/edit.js:179]: Check strict and Yoda condition

File: src/blocks/buttons-child/attributes.js [hunk 3]
```diff
@@ -125,7 +125,19 @@ const attributes = {
 	} ,
 	opensInNewTab: {
 		type: "boolean"
-	}
+	},
```
Comment (RajkiranBagal) [inline src/blocks/buttons-child/attributes.js:128]: Add these attributes in config file

Comment (vrundakansara) [reply src/blocks/buttons-child/attributes.js:128]: Done, thank you. ✅

File: src/blocks/buttons-child/edit.js (hunk 2 above)
Comment (vrundakansara) [reply src/blocks/buttons-child/edit.js:179]: Done, thank you. ✅

File: src/blocks/buttons-child/edit.js (hunk 1 above)
Comment (vrundakansara) [reply src/blocks/buttons-child/edit.js:56]: No such specific reason, I will remove it. Thanks 👍

**Pull request merged**